README.md

```diff
@@ -39,11 +39,11 @@ The `SITEMAP` setting must be a Python dictionary and can contain these keys:
 
     Valid frequency values are `always`, `hourly`, `daily`, `weekly`, `monthly`, `yearly` and `never`.
 
-You can exclude URLs from being included in the site map via regular expressions. For example, to exclude all URLs containing `tag/` or `category/` you can use the following `SITEMAP` setting.
+You can exclude URLs from being included in the site map via regular expressions. For example, to exclude all URLs that start with `/noindex/`, contain `/tag/` anywhere, or end with `.json` you can use the following `SITEMAP` setting:
 
 ```python
 SITEMAP = {
-    "exclude": ["tag/", "category/"]
+    "exclude": ["^/noindex/", "/tag/", "\.json$"]
 }
 ```
```

Comment:
Seems like this section should be an entry in the list above explaining options, but for `exclude`. And it might be a bit better to inline the explanations

```python
SITEMAP = {
    "exclude": [
        "^/noindex/",  # starts with '/noindex/'
        "/tag/",       # contains '/tag/'
        "\.json$",     # ends with '.json'
    ]
}
```

Reply:
Besides this, looks good to me.

Reply:
rebased on `main` and updated with this suggestion